addons/website_forum/static/src/js/website_forum.js

```diff
@@ -24,10 +24,7 @@ publicWidget.registry.websiteForum = publicWidget.Widget.extend({
         'click .vote_up:not(.karma_required), .vote_down:not(.karma_required)': '_onVotePostClick',
         'click .o_js_validation_queue a[href*="/validate"]': '_onValidationQueueClick',
         'click .accept_answer:not(.karma_required)': '_onAcceptAnswerClick',
-        'click .validate_answer [data-href]': '_onAcceptAnswerClick',
-        'mouseenter .validate_answer [data-href]': '_onRemoveValidAnswerMouse',
```

Comment:
IS there any reason to remove this feature ?https://github.com/odoo/odoo/commit/2c029a0cf64dc8b8aaf784659c53f544119c4e66

Reply:
seems no more needed, you already reimplement a similar code

just need to remove function too